packages/workflow/src/records/handler/create.ts

```diff
@@ -289,7 +289,6 @@ export default async function createRecordHandler(
   // don't forward hospital notifications
   const notificationDisabled =
     isHospitalNotification(record) ||
-    event === EVENT_TYPE.MARRIAGE ||
     eventAction === 'sent-for-approval' ||
```

Comment:
We don't want to disable notification for 'sent-for-approval' no?

Reply:
Thought i fixed it in a previous commit!